cleanup_all_covid_calls.py

```diff
@@ -0,0 +1,130 @@
+import pandas as pd
+import numpy as np
+from datetime import datetime
+from utils import (
+    explode_needs,
+    get_lat,
+    get_lng,
+    replacements,
+    write_output_file,
+)
+
+pd.options.mode.chained_assignment = None
+
+
+def cleanup(dfs):
+    ### Cleanup for All COVID Calls dashboard
+
+    # step 1
+    # select required columns from VIA LINK’s Disaster Form
+    # pretty sure the distaster form is "Uncleaned data type 1 VIA LINK"
+    VIA_LINK_REQUIRED_COLUMNS_DISASTER = [
+        "CallReportNum",
+        "ReportVersion",
+        "CallDateAndTimeStart",
+        "CityName",
+        "CountyName",
+        "StateProvince",
+        "PostalCode",
+        "Client Information - Age Group",
+        "Client Information - Call Type",
+        "Client Information - Identifies as",
+        "Concerns/Needs - Concerns/Needs",
+        "Contact Source - Program ",  # ending space is needed
+        "Needs - Basic Needs Requested",
+    ]
+    vialink1_df = dfs["Uncleaned data type 1 VIA LINK"][
```

Comment:
This key name should be a configurable value.  Also, we should determine the expected input format first.  Maybe this won't be a xlsx file but instead 2 separate csv files